draft-ietf-rats-corim.md

```diff
@@ -1080,84 +1080,112 @@ The verification procedure is divided into three separate phases:
 
-For example, if the Evidence format is known in advance, tags that do not match the expected profile can be readily discarded.
+For example, if the Evidence format is known in advance, tags that do not match
+the expected profile can be readily discarded.
 
 The selection process MUST yield at least one usable tag.
 
 ### CoBOM Extraction
 
-All the available Concise Bill Of Material (CoBOMs) tags are then collected from the selected CoRIMs.
+All the available Concise Bill Of Material (CoBOMs) tags are then collected
+from the selected CoRIMs.
 
 The verifier MUST activate all tags referenced by a CoBOM.
 
-After the verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
+After the verifier has processed all CoBOMs it MUST discard any tags which have
+not been activated by a CoBOM.
 
 ### Tags Identification and Validation
 
-The verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}), Concise Software ID Tags (CoSWID, {{-coswid}}),
-and/or Concise Trust Anchor Stores (CoTS, {{?I-D.ietf-rats-concise-ta-stores}}) -- from the selected CoRIMs.
+The verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}),
+Concise Software ID Tags (CoSWID, {{-coswid}}),
+and/or Concise Trust Anchor Stores (CoTS, {{?I-D.ietf-rats-concise-ta-stores}}) --
+from the selected CoRIMs.
 
-The verifier MUST discard all tags which are not syntactically and semantically valid.
-In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples) MUST be successfully resolved.
+The verifier MUST discard all tags which are not syntactically and semantically
+valid.
+In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples)
+MUST be successfully resolved.
 
 ### Appraisal Context Construction
 
 All of the validated and potentially useful tags are loaded into the Appraisal Context. 
-Each tag is loaded into the Appraisal Context as a single unit, later stages will accept or ignore the whole tag.
+Each tag is loaded into the Appraisal Context as a single unit, later stages
+will accept or ignore the whole tag.
 
 This concludes the initialisation phase.
 
 ## Evidence Collection
 
-In the evidence collection phase the verifier communicates with attesters to collect evidence.
+In the evidence collection phase the verifier communicates with attesters to
+collect evidence.
 
-The first part of the Evidence collection phase does not perform any cryptographic validation.
+The first part of the Evidence collection phase does not perform any
+cryptographic validation.
 This allows verifiers to use untrusted code for their initial Evidence collection.
 
-The results of the evidence collection are protocol specific data and transcripts which can be used as input to appraisal by a trusted Verifier.
+The results of the evidence collection are protocol specific data and transcripts
+which can be used as input to appraisal by a trusted Verifier.
```

Comment:
streamline

```suggestion
which are used as input to appraisal by the Verifier.
```

Reply:
'can be' seems appropriate since Evidence can be protected by a conveyance protocol or by a conveyance mechanism such as COSE wrapper.